modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/util/LabelItem.java

```diff
@@ -73,7 +75,10 @@ public void setLarge(boolean large) {
 	}
 
 	public void setStatus(int status) {
-		setLabel(WorkflowConstants.getStatusLabel(status));
+		setLabel(
+			LanguageUtil.get(
```

Comment:
This feels like doing too much in something that was otherwise a simple bag of attributes. Wondering if we're going to find ourselves in a place where we'd rather have the non-localized value here.

Specially confusing is the fact that it was showing localized but in english. Was that really the case, or was it simply showing the `key`? If it was the case... who was localizing it? Maybe the bug is somewhere else that's not using `mostRelevantLocale` to localize?

Reply:
Hi @jbalsas ,
The label showed seems to be the english translation but it is actually the string returned by method `WorkflowConstants.getStatusLabel`. This method does not perform any translation, it returns the language key that it just happens to be the word in english.

I know that this change may seem a little out of place but I put the code in this class partly because of this commit a54d69eccae9b8b655f852d017ccf090c0478f9a in which something that looks very similar has been already comitted. The other reason is that class WorkflowConstants does not contain any logic either so I could not find a better place to put it

Reply:
> The label showed seems to be the english translation but it is actually the string returned by method [...]

That was my guess, thanks for confirming! Might be good to make that distinction also clear in the bug reports in the future, because it's not the same thing ;)

> I know that this change may seem a little out of place but I put the code in this class partly because of this commit a54d69e in which something that looks very similar has been already comitted. The other reason is that class WorkflowConstants does not contain any logic either so I could not find a better place to put it

Fair enough, thanks for pointing that one out!